prov/efa/src/efa_domain.h

```diff
@@ -71,10 +72,18 @@ static inline bool efa_is_cache_available(struct efa_domain *efa_domain)
  * @return	a string to be append to domain name
  */
 static inline
-const char *efa_domain_name_suffix(enum fi_ep_type ep_type)
+const char *efa_domain_name_suffix(enum efa_ep_type ep_type)
 {
-	assert(ep_type == FI_EP_RDM || ep_type == FI_EP_DGRAM);
-	return (ep_type == FI_EP_RDM) ? "-rdm" : "-dgrm";
+	switch (ep_type) {
+		case EFA_EP_RDM:
+			return "-rdm";
```

Comment:
It is a bit odd to me, I thought both direct and non-direct domain can have the same domain name.

Reply:
Will change domain name suffix for efa-direct to `-rdm`